SystemManager/Src/SystemManager.cpp

```diff
@@ -0,0 +1,43 @@
+//
+// Created by Gagan Deep Singh on 2023-07-08.
+// Edited by Navtaj Hundal on 2023-07-23.
+//
+
+#include "SystemManager.hpp"
+#include "sbus_receiver.hpp"
+#include "sbus_defines.h"
+#include "rcreceiver_datatypes.h"
+#include "drivers_config.hpp"
+#include "independent_watchdog.h"
+#include "tim.h"
+
+SystemManager::SystemManager():
+    rcController_(&huart2),
+    throttleMotorChannel_(&htim2, TIM_CHANNEL_1),
+    yawMotorChannel_(&htim2, TIM_CHANNEL_2),
+    rollMotorChannel_(&htim2, TIM_CHANNEL_3),
+    pitchMotorChannel_(&htim2, TIM_CHANNEL_4),
+    watchdog_(&hiwdg)
```

Comment:
I'm fine with the way this is setup, but it defeats the purpose of the model config system so I'm confused why you need to be blocked on the including the model config as part of this  PR?

Reply:
I'll revert the model config changes for this one, forgot to undo those from before lol

Reply:
@TheSpaceDragon has this been changed?

Reply:
The model config changes have been reverted now. These are directly interfacing with the IOC and drivers. I believe the plan is to implement this with model-config for later PRs